modules/op-about-tasks.adoc

```diff
@@ -46,3 +46,10 @@ spec: <4>
 This Task starts the pod and runs a container inside that pod using the `maven:3.6.0-jdk-8-slim` image to run the specified commands. It receives an input directory called `workspace-git` that contains the source code of the application.
 
 The Task only declares the placeholder for the Git repository, it does not specify which Git repository to use. This allows Tasks to be reusable for multiple Pipelines and purposes.
+
+[WARNING]
+====
+{pipelines-title} 1.3 and earlier versions in Technology Preview (TP) create a task without verifying the xref:../authentication/managing-security-context-constraints.adoc[Security Context Constraints] (SCC) of users. This allows any authenticated user to create a task with a container running with a privileged SCC.
+
+To avoid such security issues in the production scenario, do not use {pipelines-shortname} versions that are in TP. Instead, consider upgrading the operator to {pipelines-shortname} 1.4 or later versions that are in General Availability.
```

Comment:
Operator - use title case
Suggest: that are in General Availability./that are generally available.
Alternatively reword something like: Instead, consider upgrading the Operator to generally available versions such as {pipelines-shortname} 1.4 or later.